server/app/policies/library_entry_policy.rb

```diff
@@ -10,10 +10,13 @@ def show?
     true
   end
 
+  def create?
+    !user.nil?
+  end
```

Comment:
Hum… In this specific case, can't we just auto assign the user_id to the current_user on a before_crate filter?